apps/condo/domains/news/schema/NewsItem.js

```diff
@@ -65,6 +65,13 @@ const NewsItem = new GQLListSchema('NewsItem', {
             type: 'DateTimeUtc',
         },
 
+        scopes: {
```

Comment:
Is migration missing here ?

Reply:
No migration needed. Created fields are enough.